scrooge-generator/src/main/resources/nodegen/service.mustache

```diff
@@ -90,17 +89,16 @@ export class Client {
 }
 
 export class Processor {
-    private _handler: any
+    private _handler
```

Comment:
why remove the type declaration?  It would be best to define a function handler type and use it.  Also why is this not defined with a let?